src/components/views/read/cvForm/FormLayout.tsx

```diff
@@ -0,0 +1,327 @@
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is personal info
+              </p>
+            )}
+            {activeStep === 2 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is education
+              </p>
+            )}
+            {activeStep === 3 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is work experience
+              </p>
+            )}
+            {activeStep === 4 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is skills
+              </p>
+            )}
+            {activeStep === 5 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is portfolio
+              </p>
+            )}
+            {activeStep === 6 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is personal motivation
+              </p>
+            )}
+            {activeStep === 7 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is reference
+              </p>
+            )}
+            {activeStep === 8 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is template
+              </p>
+            )}
+            {activeStep === 9 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is review
+              </p>
+            )}
+          </div>
+          <div
+            style={{
+              alignSelf: 'stretch',
+              position: 'relative',
+              height: '194px',
+              display: 'none',
```

Comment:
Why does this div need to not be displayed ?

Reply:
This code implementation of the design is generated by an extension from Figma, and it generated a lot of messy divs' and not necessary lines, but I digged the neccessary part.